### PR TITLE
Updating racon to 1.5.0

### PR DESCRIPTION
**Pull request by erinyoung**
I don't use racon very much, but when I do I like using the most-current version - which isn't in this repo.

Update: I have added a builder test and downloaded some test files from nf-core for some testing.

I have made a few changed to the prior version of racon (https://github.com/StaPH-B/docker-builds/blob/master/racon/1.4.3/Dockerfile). 

Version 1.5.0 updates 
- the base image is now jammy
- there are now builder, app, and test stages
- changed the repository to https://github.com/lbcb-sci/racon
  - added this repo's license to racon's line on Program_licenses.md

~~This isn't ready for prime-time yet, but I've started the process.~~

~~My TODO LIST:~~
~~- add in a builder stage~~
~~- add in a real test~~

Pull Request (PR) checklist:
- [X] Include a description of what is in this pull request in this message.
- [X] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [X] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [X] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [X] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [X] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [X] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [X] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [X] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

**Comment by erinyoung** [comment]
Closes https://github.com/StaPH-B/docker-builds/issues/741

File: racon/1.5.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+LABEL software="Racon"
+LABEL software.version="${RACON_VER}"
+LABEL description="Long read assembly and polishing tools"
+LABEL website="https://github.com/lbcb-sci/racon"
+LABEL license="https://github.com/lbcb-sci/racon/blob/master/LICENSE"
+LABEL maintainer="Erin Young"
+LABEL maintainer.email="eriny@utah.gov"
+
+# install python
+RUN apt-get update && apt-get install -y \
```
Comment (Kincekara) [inline racon/1.5.0/Dockerfile:52]: Did you forget to use "--no-install-recommends" or is it intentional?

**Review by Kincekara — APPROVED** [review]
Looks great!

**Comment by Kincekara** [comment]
Thank you for the update and modernization!
You can check the image deployment from here: https://github.com/StaPH-B/docker-builds/actions/runs/7815886827.

**Pull request merged**